stdlib/lang-parser/parserModule/ast.bal

```diff
@@ -0,0 +1,200 @@
+import ballerina/io;
```

Comment:
Add license header to all the files.

Reply:
license header added to all files except for the test resource files (because the parser doesn't support comments yet).